README.md

```diff
@@ -24,9 +24,8 @@ export PATH=${GOPATH}/bin:${PATH}
 And then you can check out `github.com/Netflix/titus-executor` to `${GOPATH}/src/github.com/Netflix/titus-executor` -- an easy way to do this is go run `go get -u github.com/Netflix/titus-executor`.
 
 # Initial setup steps
-1. go get -u github.com/alecthomas/gometalinter
-2. gometalinter --install
-3. go get -u github.com/kardianos/govendor
+1. `./hack/install-gometalinter.sh`
+2. go get -u github.com/kardianos/govendor
```

Comment:
We install this, via the makefile automagically, now, don't we?

Reply:
Yes - removed this line.